cookiecutter/repository.py

```diff
@@ -11,12 +11,11 @@
 from .zipfile import unzip
 
 REPO_REGEX = re.compile(r"""
-(?x)
 ((((git|hg)\+)?(git|ssh|https?):(//)?)  # something like git:// ssh:// etc.
  |                                      # or
  (\w+@[\w\.]+)                          # something like user@...
 )
-""")
+""", flags=re.X)
```

Comment:
```suggestion
""", flags=re.VERBOSE)
```
would be nice if you forget what `X` is.